eth/vm/computation.py

```diff
@@ -320,6 +324,17 @@ def stack_push_int(self) -> Callable[[int], None]:
     @cached_property
     def stack_push_bytes(self) -> Callable[[bytes], None]:
         return self._stack.push_bytes
+    
+    #
+    # Return Stack Management
+    #
+    @cached_property
+    def rstack_push_int(self) -> Callable[[int], None]:
```

Comment:
These will need to get propogated down to the `evm.abc.ComputationAPI` class where the general interface is defined.